weave

```diff
@@ -547,16 +547,20 @@ http_call() {
         "$1 container is not present. Have you launched it?" \
         "$1 container is not running." \
         || return 1
+    HTTP_CALL_CONTAINER=$1
     shift 1
-    http_call_ip $CONTAINER_IP "$@"
+    if ! http_call_ip $CONTAINER_IP "$@" ; then
+        echo "Call to $HTTP_CALL_CONTAINER failed." >&2
+        return 1
+    fi
 }
 
 call_weave() {
-    http_call $CONTAINER_NAME $HTTP_PORT "$@"
+    http_call $CONTAINER_NAME $HTTP_PORT "$@" --show-error
```

Comment:
You actually need both `-s -S` to get (no progress messages) and (error messages)